pkgs/development/compilers/scryer-prolog/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, gnum4
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "scryer-prolog";
+  version = "0.8.117-dev";
```

Comment:
Can we use the `v0.8.127` tag instead of this dev release? We prefer to have tagged releases in nixpkgs.

Reply:
Done.